monasca-alarms/monasca_alarm_definition.py

```diff
@@ -207,9 +184,15 @@ def run(self, data_file):
 
         yaml_data = yaml.safe_load(yaml_text)
 
-        self._keystone_auth()
-        self._monasca = client.Client(self._args['api_version'], self._api_url, token=self._token)
-        self._print_message('Using Monasca at {}'.format(self._api_url))
+        if self._args['monasca_api_url'] is None:
```

Comment:
That's a bit incorrect, but not here but in client - wonder if shouldn't support endpoint retrieving in client. But that's a note that does not change anything in here.